src/_data/core.yml

```diff
@@ -0,0 +1,54 @@
+navigation:
+  top:
+    - caption: Blog
+      dropdown: categories
+      url: "/blog/"
+    - caption: Services
+      url: "/services/"
+    - caption: Work
+      url: "/work/" # note: redundant.. link found in blog dropdown also
+    - caption: Team
+      url: "/team/"
+  footer:
+    expertise: # todo: make this like services.yml
+      - caption: Docker
+      - caption: Elixir / Phoenix
+      - caption: Ruby
```

Comment:
Let's just make this: Ruby on Rails

That's what most people are looking for when they're Googling.